proxy/tests/support/mod.rs

```diff
@@ -30,6 +30,48 @@ use self::tower_h2::{Body, RecvBody};
 use std::net::SocketAddr;
 pub use std::time::Duration;
 
+/// Environment variable for overriding the test patience.
+pub const ENV_TEST_PATIENCE_MS: &'static str = "RUST_TEST_PATIENCE_MS";
+pub const DEFAULT_TEST_PATIENCE: u64 = 15;
```

Comment:
Perhaps just make this `Duration` now that `Duration::from_millis` is `const`?

Reply:
Is that on stable?

Reply:
as of 1.25, yes!